C7_ball_launcher.md

```diff
@@ -0,0 +1,86 @@
+# More info: ball launcher
+
+The robot laboratory has a ball launcher that can be activated programmatically.
+
+## How to
+
+### Start the ball launcher
+
+The ball launcher is connected to a multiple socker power supply. Turn it on. The raspberry pi attached to the launcher should start and display something like:
+
+```
+-----------------------------------
+| Welcome to ball launcher beepy  |
+-----------------------------------
+
+to ssh to me: user pi, password: ---
+
+- sourcing colcon workspace
+- killing already running server
+- starting the ball launcher server on IP 10.42.31.174 and PORT 5555
+sleep() takes 1 positional argument but 2 were given
```

Comment:
Should this really be displayed? :)

Reply:
indeed ! I updated to hide this.

Reply:
oh, now not sure if you meant in the doc or on the launcher screen.
Agree it should the IP should not be the on the doc, so it now hidden.
For the launcher screen, I think it is very convenient to get the password and other information easily. The set of people having credentials to get into the lab seems low enough.

Reply:
I actually only meant the "sleep() takes 1 positional argument but 2 were given" line, this looks like it is an error in the launcher screen software?

Reply:
Actually this is an error in the launcher server. Looks like it work anyway.